packages/ui/src/components/Button.svelte

```diff
@@ -16,16 +16,17 @@
   import type { Asset, IntlString } from '@anticrm/platform'
   import { onMount } from 'svelte'
   import { registerFocus } from '../focus'
-  import type { AnySvelteComponent, ButtonKind, ButtonSize } from '../types'
+  import type { AnySvelteComponent, ButtonKind, ButtonShape, ButtonSize } from '../types'
   import Icon from './Icon.svelte'
   import Label from './Label.svelte'
   import Spinner from './Spinner.svelte'
 
   export let label: IntlString | undefined = undefined
+  export let text: string | undefined = undefined
```

Comment:
Why we need text here? Please use IntlString with parameters if you need to put some text inside and pass value using labelParams.

Reply:
Done.